BeVolt/App/src/main.c

```diff
@@ -231,7 +234,7 @@ void faultCondition(void){
 // E.g. If you want to run a LTC6811 test, change "#define CHANGE_THIS_TO_TEST_NAME" to the
 //		following:
 //		#define LTC6811_TEST
-#define CLI_TEST
+#define DashboardMessage
```

Comment:
Definitions are best used in all uppercase to easily identify. I suggest renaming to something like this.
```suggestion
#define DASHBOARD_MSG
```